### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Scotty1luv** [community/community#50958] [New to GitHub]
I need help

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @Scotty1luv  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
